### ISSUE TITLE
Bug Report

### ISSUE THREAD

**Issue by iantiuri** [Dans-Plugins/Medieval-Factions#1698]
Can't seem to create a new functional gate, the "/gate create [name]" commands seems to activate, and It seems that it actually recognizes there is supposed to be a gate in the area I assign it to be, causing the blocks not to be able to be destroyed.
But using the trigger of the gate just doesn't do anything.
A gate I created in a previous update of MF4 seems to work just fine.

## Java version
No clue

**Plugin version**
5.1.2

PaperMC
1.19.2

**Comment by alyphen** [comment]
Are you powering the trigger block with redstone?
If you set the trigger to be the lever, it may be that it should be the block it is connected to instead.

**Comment by iantiuri** [comment]
I didn't get at first what you meant, but I got it now.
I didn't know you had to assign a block and power that block with redstone for it to work.
Especially since the gate tutorial video on Spigot still shows the old way to do it.
In MF4 you assigned the trigger directly to a lever/button.
Didn't know that it was changed in MF5.

Guess it works as intended just didn't expect it was supposed to function that way.
Thanks for the help!
